util/cmd/cmd.go

```diff
@@ -49,13 +49,17 @@ func healthHandler(pulse Pulse) func(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// AddUnhealthyLogger adds a periodic poller which reports an unhealthy status as a log message.
-func AddUnhealthyLogger(pulse Pulse, pollInterval time.Duration) {
+const pollInterval = time.Second
+
+// AddUnhealthyLogger adds a periodic poller which reports an unhealthy status.
+// The healthCounter is increased by pollInterval if unhealthy.
+func AddUnhealthyLogger(pulse Pulse, unhealthyCounter prometheus.Counter) {
 	go func() {
 		healthy := true
 		tickCh := time.Tick(pollInterval)
 		for range tickCh {
 			if err := pulse.Health(); err != nil {
+				unhealthyCounter.Add(pollInterval.Seconds())
```

Comment:
Because it's the time unhealthy. If you just increase by 1, there is no way to know how long it was unhealthy unless you know how often this poller updates (which might slow down under load, etc.)